src/lib/ts/preset.ts

```diff
@@ -76,3 +77,36 @@ export function decentralizedExchange(tradesPerDay: number): Feature[] {
 
   return features;
 }
+
+export function largeData(users: number): Feature[] {
+  const INGRESS_PER_USER = 1;
+  const USERS_PER_DAY = 100;
+  const STORAGE_BYTES_PER_USER = 4 * 1000 * 1000; // 4MB
```

Comment:
not sure it matters, but perhaps worth asking: here we calculate in MB (powers of 10), aren't we doing the charging in MiB (powers of 2) on the backend?

Reply:
In the UI we show MB or GB and so I followed that (but maybe we use that but mean MiB and GiB). Tbh, I don't think most people will get that small difference and it's anyway a ballpark.

Reply:
Ok, so we mean `MiB` or `GiB` in the UI even though we use `MB` and `GB`. I'll change it here to be consistent.